.golangci.yml

```diff
@@ -8,6 +8,102 @@ output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
   format: colored-line-number
 
+linters:
+  enable-all: true
+  fast: false
+  
+  disable:
+    # These linters are all deprecated. We disable them explicitly to avoid the
+    # linter logging deprecation warnings.
+    - deadcode
+    - varcheck
+    - scopelint
+    - structcheck
+    - interfacer
+    - exhaustivestruct
+    - ifshort
+    - golint
+    - maligned
+    - nosnakecase
+
+    # These are linters we'd like to enable, but that will be labor intensive to
+    # make existing code compliant.
+    - wrapcheck
+    - varnamelen
+    - testpackage
+    - paralleltest
+    - nilnil
+    - gomnd
```

Comment:
I think these would be useful, but I'm not going to tackle them in this PR. They'd require quite a lot of work to make our existing code compliant (e.g. either wrapping or `//nolint`-ing all errors).